pages/resources/index.js

```diff
@@ -156,12 +155,75 @@ export const getStaticProps = async ({ params }) => {
     return a.kicker.localeCompare(b.kicker);
   });
 
+  const wrapper = await getProps(collections, recents);
   const navigation = await getNavigation();
 
   return {
     props: {
-      collections,
       navigation,
+      wrapper,
     },
   };
 };
+
+async function getProps(collections, recents) {
+  // create a wrapper which contains all necessary data for the page
+  // includes page content based on section (content) and headings to pass to the TOC
+  // we have to hard-code What's New because it's too unique
+  let wrapper = { content: {}, headings: [] };
+  let content = { categories: [] };
+  let [name, posts, description, size, type, id] = [
+    "What's New?",
+    recents,
+    null,
+    "full",
+    "New",
+    "What's%20New?",
+  ];
+  let category = { name, posts, description, size, type, id };
+  content.categories.push(category);
+
+  let [label, level, headings] = [name, 2, []];
+  let heading = { label, level };
+  headings.push(heading);
+
+  const descriptions = {
+    Guides:
+      "Explore the processes and tools we use in our step-by-step written guides.",
+    Tutorials:
+      "Watch or read through our tutorials to understand key techniques and concepts.",
+    Workshops:
+      "Sharpen your design and development skills with our in-depth recorded workshops.",
+  };
+
+  for (let i = 0; i < collections.length; i++) {
```

Comment:
Is there a particular reason we're doing a for loop instead of mapping over the collections array?